src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/EventLogInternal.cs

```diff
@@ -1030,7 +1029,7 @@ public void ModifyOverflowPolicy(OverflowAction action, int retentionDays)
                 if (retentionDays < 1 || retentionDays > 365)
                     throw new ArgumentOutOfRangeException(SR.RentionDaysOutOfRange);
 
-                retentionvalue = (long)retentionDays * SecondsPerDay;
+                retentionvalue = (long)retentionDays * TimeSpan.SecondsPerDay;
```

Comment:
The check failure here is an issue in the pipeline, I think, because TimeSpan _now_ does contain a [`SecondsPerDay`](https://github.com/dotnet/runtime/pull/103498/files#diff-2b619d4edf7d9e9f6b5dfa45016a3a48225b939dc587ad2c7482532f7b6b9a2eR187)

Reply:
This library builds for .NET Standard and .NET Framework:
https://github.com/dotnet/runtime/blob/4834a507be8698d901ff36d10284ca8703743b93/src/libraries/System.Diagnostics.EventLog/src/System.Diagnostics.EventLog.csproj#L4

So, this new API cannot be used here (well, not without `#if`, which I imagine isn't worth it for this particular API...)

Reply:
I was worried that might be the case, I will revert this particular change as it's probably not worth an `#if` unless folks would rather I do that.